client/extensions/woocommerce/components/list/list-rows/index.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
But do we need to fully abstract out the markup at this time? Also, the style rationale is a consequence of wanting to not combine styles in a single scss for the overall list component, no?

My vote would be to not break things down this far, but if you prefer it this way, that's fine too :)